File: bin/get_tsumo_frrom_haihu.rb

#ruby
STDOUT.sync = true
Dir.chdir(File.dirname(File.expand_path(__FILE__))+"/../test_cases/workspace/")
require "../../lib/haihu/get_tsumo_from.rb"

#input_file = "sample.txt"
input_file = "../../etc/hounan2015/hounan2015.txt"
#output_file = "../../test_cases/workspace/output/output.txt"
output_file = "../../test_cases/workspace/output/hounan2015.csv"
File.open(output_file,'w'){|file| file = nil}

Get_tsumo_from::get_tsumo_from input_file, output_file

File: bin/get_tsumo_frrom_haihu.py
#!/usr/bin/python
# -*- coding: utf-8 -*-

#########################################################
haihu_files=["../../etc/data/hounan2015/hounan2015.txt"]
codec = "cp932"
#########################################################

import sys,os
sys.path.append(os.path.dirname(os.path.abspath(__file__)) + "/../lib/utils")
from log import Log as l
log = l.getLogger()


import sys,os
sys.path.append(os.path.dirname(os.path.abspath(__file__)) + "/../lib/haihu")
log.debug(os.path.dirname(os.path.abspath(__file__)) + "/../lib/haihu")
import tenhou
t = tenhou.Tenhou()

for haihu in haihu_files:
	output = t.get_tsumo_from(haihu, codec)













